CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 ### Added
 
 - Added `ErrorHandlerFunc` to use a function as an `"go.opentelemetry.io/otel".ErrorHandler`. (#2149)
+- Added `recordStackTrace` function to capture stack trace within `RecordError` API. (#2163)
```

Comment:
Please rephrase this in terms of exporter behavior that the user will have use for.